src/main/java/com/birdie/kafka/connect/json/SchemaTransformer.java

```diff
@@ -303,4 +310,17 @@ public static Object valueFromLiteralJacksonTreeNode(JsonNode node) {
 
         throw new IllegalArgumentException("Found JSON node of type '"+node.getNodeType()+"' but not supported.");
     }
+
+    private String parentSchemaName(String transformedArraySchemaName) {
```

Comment:
I couldn't find a cleaner solution for this...
We add `_array_item` to each schema name inside an array on line 107 but I need to remove it to find the name of the parent field and match it to the list of ignored fields.

Reply:
@ChloePont I prefer this option:
https://github.com/birdiecare/connect-smts/pull/26